api/client/options.proto

```diff
@@ -114,4 +114,6 @@ message CommandLineOptions {
   map<string, uint64> failure_predicates = 24;
   // See :option:`--open-loop` for details.
   google.protobuf.BoolValue open_loop = 21;
+  // See :option:`--jitter-uniform`
+  google.protobuf.Duration jitter_uniform = 25 [(validate.rules).duration.lt.seconds = 1];
```

Comment:
Why this limit? I don't think we need to be more opinonated than what the implementation dictates in the API.

Reply:
My reasoning here is that higher values may be indicative of accidental bad input, I think the usual input here is some fraction of the targetted frequency. Our minimal rps is 1/s..
Do you think we will miss usecases when disallowing >1s uniform jitter? 

ps - The seconds part of the proto can be disregarded when staying below 1 second, which is why below the conversion api isn't used.

Reply:
I'm open to changing this, otherwise I'll add a comment summarizing the reasoning here, let me know which one you prefer

Reply:
It's probably fine to limit to a second. I don't think in general though our philosophy is to prevent the user misconfiguring via PGV limits beyond the implementation preconditions; we let folks pick their own adventure, maybe they have a legitimate unforeseen need for some esoteric config value.

Reply:
fixed!